rest-service/manager_rest/rest/resources_v3/secrets.py

```diff
@@ -47,11 +54,13 @@ def get(self, key):
         else:
             # Returns the decrypted value
             try:
-                secret_dict['value'] = decrypt(secret.value)
+                secret_dict['value'] = get_secret_method(key)[1]
```

Comment:
```suggestion
                secret_dict['value'] = get_secret_method(key).value
```
let's keep using the attribute access, rather than index access, for this namedtuple. I don't really see a reason to switch to indexes here, and that's going to be more fragile (eg. if we add more fields there, or replace the SecretType with just the Secret model)